go.mod

```diff
@@ -4,14 +4,12 @@ go 1.12
 
 require (
 	github.com/VividCortex/gohistogram v1.0.0 // indirect
-	github.com/antihax/optional v0.0.0-20180407024304-ca021399b1a6
 	github.com/go-kit/kit v0.9.0
 	github.com/gorilla/mux v1.7.3
 	github.com/hashicorp/golang-lru v0.5.3 // indirect
 	github.com/moov-io/base v0.10.0-rc1.0.20190617172406-c11cd2f9d7fd
 	github.com/prometheus/client_golang v1.0.0
 	github.com/rickar/cal v1.0.1 // indirect
-	golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45
```

Comment:
Do we need to include these `go.mod` and `go.sum` changes?

Reply:
I don't think we need too but the dependencies were update when I ran go mod tidy.  I can remove.